backend/src/test/java/com/google/sps/TimeHelperTest.java

```diff
@@ -40,33 +42,33 @@ public void inputTimeToMillis() {
             /* second= */ 0,
             /* nanosecond= */ 0,
             /* zone= */ ZoneId.of("US/Eastern"));
-    Assert.assertEquals(/* expected */ dateTime, /* actual */ TimeHelper.getNewTimeToday(14, 0));
+    assertThat(dateTime).isEqualTo(TimeHelper.getNewTimeToday(14, 0));
 
     // 1/1/2020 10:30am ET
     dateTime = dateTime.withHour(10).withMinute(30);
-    Assert.assertEquals(dateTime, TimeHelper.getNewTimeToday(10, 30));
+    assertThat(dateTime).isEqualTo(TimeHelper.getNewTimeToday(10, 30));
 
     // 1/1/2020 11:45pm ET
     dateTime = dateTime.withHour(23).withMinute(45);
-    Assert.assertEquals(dateTime, TimeHelper.getNewTimeToday(23, 45));
+    assertThat(dateTime).isEqualTo(TimeHelper.getNewTimeToday(23, 45));
```

Comment:
thanks for converting to truth. Now it's clearer than actual and expected have been flipped. Putting the actual value (the one under test) and the expected value in the right place makes error logs easier to read. 

dateTime is the expected value whereas TimeHelper.getNEwTimeToday(23, 45) is the actual value.